website/www/site/content/en/documentation/ml/about-ml.md

```diff
@@ -38,38 +40,48 @@ limitations under the License.
+To keep your model up to date and performing well as your data grows and evolves, run these steps multiple times. In addition, you can apply ML ops to your project to automate the AI/ML workflows throughout the model and data lifecycle. Use orchestrators to automate this flow and to handle the transition between the different building blocks in your project.
 
 ## Use RunInference
 
-The recommended way to implement inference in Apache Beam is by using the [RunInference API](/documentation/sdks/python-machine-learning/). RunInference takes advantage of existing Apache Beam concepts, such as the `BatchElements` transform and the `Shared` class, to enable you to use models in your pipelines to create transforms optimized for machine learning inferences. The ability to create arbitrarily complex workflow graphs also allows you to build multi-model pipelines.
+The [RunInference API](/documentation/sdks/python-machine-learning/) is a `PTransform` optimized for machine learning inferences that lets you efficiently use ML models in your pipelines. The API includes the following features:
 
-You can integrate your model in your pipeline by using the corresponding model handlers. A `ModelHandler` is an object that wraps the underlying model and allows you to configure its parameters. Model handlers are available for PyTorch, scikit-learn, and TensorFlow. Examples of how to use RunInference for PyTorch, scikit-learn, and TensorFlow are shown in this [notebook](https://github.com/apache/beam/blob/master/examples/notebooks/beam-ml/run_inference_pytorch_tensorflow_sklearn.ipynb).
+- To efficiently feed your model, dynamically batches inputs based on pipeline throughput.
+- To optimize your pipeline for ML inference, takes advantage of existing Apache Beam concepts, such as the `BatchElements` transform and the `Shared` class.
+- To balance memory and throughput usage, determines the optimal number of models to load using a central model manager.
+- Ensures that your pipeline uses the most recently deployed version of your model with the [Automatic model refresh](#automatic-model-refresh) feature.
+- Supports [multiple frameworks and model hubs](#use-pre-trained-models), including Tensorflow, Pytorch, Sklearn, XGBoost, Hugging Face, TensorFlow Hub, Vertex AI, TensorRT, and ONNX.
+- Supports arbitrary frameworks using a [custom model handler](#use-custom-models).
+- Supports [multi-model pipelines](#multi-model-pipelines).
+- Lets you [use GPUs](/documentation/ml/runinference-metrics) to increase inference speed. Because GPUs can process multiple computations simultaneously, they are optimized for training artificial intelligence and deep learning models.
```

Comment:
This link doesn't really fit here, maybe we just cut it entirely? GPUs are runner dependent and all the docs are runner specific as a result, so maybe we could say something like "Lets you use GPUs on supported runners like Dataflow to increase inference speed..." and link to the Dataflow GPU docs?

Reply:
I edited this and added a link to the Dataflow GPU docs.